devenv/docker/blocks/loki-promtail/promtail-config.yaml

```diff
@@ -22,4 +22,4 @@ scrape_configs:
       - localhost
     labels:
       job: grafana
-      __path__: /var/log/grafana/*log
\ No newline at end of file
+      __path__: /var/log/grafana/*log
```

Comment:
sorry, is this change intentional?

Reply:
no, this was GoLand. Not sure why it did this

Reply:
should probably have a newline at the end of the file anyways